_directory/data/staabm/data.yml

```diff
@@ -0,0 +1,29 @@
+---
+# name (required)
+# may be your real name or your nick
+name: Markus Staab
+
+# geo location (required)
+# provide your favored level of detail by number of digits (like 50.107811 vs 50.11)
+# you may use services like openstreetmap.org to find out your geo location
+latitude: 50.0804
+longitude: 9.2014
+
+# bio
+# describe yourself, your involvement in the project and the community or whatever is relevant for you.
```

Comment:
@schuer actually I am not sure, whether we should repeat all those "docs" inside each user file.

in case we need to change any of it, it will become a lot of work. maybe just add a link to the actual example file at the top?

Reply:
@staabm You’re right. However, it wouldn’t be a big issue if user entries contain wrong/old »docs«. People are even allowed to leave them out, add notes or whatever they like to do. So we could say: no worries about the docs, we don’t care, it’s up to the users to decide about their docs.